runelite-client/src/main/java/net/runelite/client/plugins/worldmap/TransportationPointLocation.java

```diff
@@ -0,0 +1,199 @@
+	DRAGONTOOTH_TO_PHASMATYS("Ship to - Port Phasmatys", new WorldPoint(3791, 3561, 0)),
+	DIGSITE_TO_FOSSIL("Ship to - Fossil island", new WorldPoint(3361, 3448, 0)),
+	ENTRANA_TO_PORTSARIM("Ship to - Port Sarim", new WorldPoint(2833, 3334, 0)),
+	FISHINGPLAT_TO_ARDOUGNE("Ship to - Ardougne", new WorldPoint(2779, 3271, 0)),
+	HARMLESS_TO_PORT_PHASMATYS("Ship to - Port Phasmatys", new WorldPoint(3682, 2951, 0)),
+	ICEBERG_TO_RELLEKKA("Ship to - Rellekka", new WorldPoint(2657, 3988, 0)),
+	ISLAND_TO_APETOLL("Ship to - Ape Toll", new WorldPoint(2891, 2726, 0)),
+	JATIZZO_TO_RELLEKKA("Ship to - Rellekka", new WorldPoint(2420, 3780, 0)),
+	KARAMJA_TO_PORT_SARIM("Ship to - Port Sarim", new WorldPoint(2955, 3144, 0)),
+	KARAMJA_TO_KHAZARD_PORT("Ship to - Khazard Port", new WorldPoint(2763, 2957, 0)),
+	KHAZARD_TO_SHILO("Ship to - Shilo", new WorldPoint(2673, 3143, 0)),
+	LANDSEND_TO_PORTSARIM_PORTPISCARILLUS("Ship to Port Sarim/Port Piscarillus", new WorldPoint(1503, 3398, 0)),
+	LUNAR_ISLE_TO_PIRATES_COVE("Ship to - Pirates Cove", new WorldPoint(2137, 3899, 0)),
+	MISCELLANIA_TO_RELLEKKA("Ship to - Rellekka", new WorldPoint(2579, 3846, 0)),
+	NEITZNOT_TO_RELLEKKA("Ship to - Rellekka", new WorldPoint(2310, 3779, 0)),
+	PESTCONTROL_TO_PORTSARIM("Ship to - Port Sarim", new WorldPoint(2659, 2675, 0)),
+	PIRATES_COVE_TO_LUNAR_ISLE("Ship to - Lunar isle", new WorldPoint(2223, 3796, 0)),
+	PIRATES_COVER_TO_RELLEKKA("Ship to - Rellekka", new WorldPoint(2212, 3794, 0)),
+	PORT_PHASMATYS_TO_DRAGONTOOTH("Ship to - Dragontooth island", new WorldPoint(3703, 3487, 0)),
+	PORT_PHASMATYS_TO_HARMLESS("Ship to - Mos le Harmless", new WorldPoint(3709, 3497, 0)),
+	PORT_PISCARILIUS_TO_PORTSARIM_lANDSEND("Ship to - Port Sarim/Lands end", new WorldPoint(1823, 3692, 0)),
+	PORTSARIM_TO_GREAT_KOUREND("Ship to - Great Kourend", new WorldPoint(3054, 3244, 0)),
+	PORTSARIM_TO_ENTRANA("Ship to - Entrana", new WorldPoint(3046, 3233, 0)),
+	PORTSARIM_TO_KARAMJA("Ship to - Karamja", new WorldPoint(3029, 3218, 0)),
+	PORTSARIM_TO_CRANDOR("Ship to - Crandor", new WorldPoint(3045, 3205, 0)),
+	PORTSARIM_TO_PEST_CONTROL("Ship to - Pest Control", new WorldPoint(3039, 3201, 0)),
+	RELLEKKA_TO_JATIZZO_NEITIZNOT("Ship to - Jatizzo/Neitiznot", new WorldPoint(2639, 3710, 0)),
+	RELLEKKA_TO_MISCELLANIA("Ship to - Miscellania", new WorldPoint(2627, 3692, 0)),
+	RELLEKKA_TO_WATERBIRTH("Ship to - Waterbirth", new WorldPoint(2621, 3683, 0)),
+	RELLEKKA_TO_WEISS_ICEBERG("Ship to - Weiss/Iceberg", new WorldPoint(2707, 3735, 0)),
+	RELLEKKA_TO_UNGAEL("Ship to - Ungael", new WorldPoint(2638, 3698, 0)),
+	RIMMINGTON_TO_CORSAIR_COVE("Ship to - Corsair Cove", new WorldPoint(2909, 3227, 0 )),
+	WATERBIRTH_TO_RELLEKKA("Ship to - Rellekka", new WorldPoint(2549, 3758, 0)),
+	WEISS_TO_RELLEKKA("Ship to - Rellekka", new WorldPoint(2847, 3967, 0)),
+	UNGAEL_TO_RELLEKKA("Ship to - Rellekka", new WorldPoint(2276, 4034, 0)),
+
+	//Row Boats
```

Comment:
Might be worth adding the destinations to these.

Reply:
There are multiple destinations for most of them. That's why destinations weren`t included.

Reply:
It seems most of them have only 1 destination, with only the Shayzien/Molch/Molch Island one and one of the Mushroom meadow boats having 2, at least of the top of my head.

Reply:
So do you want them all the destinations for all the rowboats than?

Reply:
Yeah, makes them more useful